src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetMvcIntegration.cs

```diff
@@ -96,6 +97,14 @@ public static Scope CreateScope(dynamic controllerContext)
                 span.SetTag(Tags.AspNetRoute, route?.Url);
                 span.SetTag(Tags.AspNetController, controllerName);
                 span.SetTag(Tags.AspNetAction, actionName);
+
+                if (propagatedContext != null)
```

Comment:
This feels like it is going to suck to have to include in every integration.

Not sure I have a good idea on how to address, and not the most important thing to address at the moment.

Reply:
Agreed. I thought about that and tried to do it outside of the intergration, but it was tricky to determine if a `SpanContext` was propagated (created from incoming http headers) or not, and we need to do it after the `Span` is created so we can access the `TraceContext`. I have a few ideas, though.... coming soon!

Reply:
I would say, this is not high priority to get correct right now. This is something that we can come back to whenever in the future.

It isn't a big deal right now because there are a limited number of integrations this is being done it, copy/pasta between 3-4 integrations today is ok (not ideal of course, but ok).